src/java.base/share/classes/jdk/internal/module/ModulePatcher.java

```diff
@@ -131,14 +131,15 @@ public ModuleReference patchIfNeeded(ModuleReference mref) {
 
                     // exploded directory without following sym links
                     Path top = file;
-                    Files.find(top, Integer.MAX_VALUE,
-                               ((path, attrs) -> attrs.isRegularFile()))
-                            .filter(path -> (!isAutomatic
-                                    || path.toString().endsWith(".class"))
-                                    && !isHidden(path))
+                    try (Stream<Path> stream = Files.find(top, Integer.MAX_VALUE,
+                            ((path, attrs) -> attrs.isRegularFile()))) {
+                        stream.filter(path -> (!isAutomatic
+                            || path.toString().endsWith(".class"))
+                            && !isHidden(path))
```

Comment:
Trivially, can the indentation of the two boolean operators be pushed in so that they align underneath the start of initial boolean expression.